File: mobile_through_project_final/src/main/java/com/ssafy/cafe/model/dto/Order.java
package com.ssafy.cafe.model.dto;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Order {
    private Integer id;
    private String userId;
    private String orderTable;
    private Date orderTime;

    private Character completed;
    private List<OrderDetail> details ;
    private Stamp stamp;
    
    @Builder
    public Order(Integer id, String userId, String orderTable, Date orderTime, Character complited) {
        super();
        this.id = id;
        this.userId = userId;
        this.orderTable = orderTable;
        this.orderTime = orderTime;
        this.completed = complited;
    }

    public Order(String userId, String orderTable, Date orderTime, Character complited) {
        this.userId = userId;
        this.orderTable = orderTable;
        this.orderTime = orderTime;
        this.completed = complited;
    }
    
}

File: app/src/main/java/com/ssafy/cobaltcoffee/dto/Order.kt
package com.ssafy.cobaltcoffee.dto

import java.util.Date

class Order(
    val id: Int,
    var userId: String,
    var orderTable: String,
    val orderTime: Date,
    var completed: Char,
    var details: List<OrderDetail>,
    val stamp: Stamp
) : java.io.Serializable {
    constructor(): this(0, "", "", Date(), ' ', listOf<OrderDetail>(), Stamp())
}